jmeter_run_all.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash +x
+
+# Scenario name is prefix added to the log files. So add something meaningful so that you can track the output easily. eg: saml_redirect_binding.
+scenario=$1
+
+# Location of the jmeter script.
+script_location=$2
+
+usage_msg="Usage: ./jmeter_run_all.sh <scenario_name> <jmeter_script_location>"
+usage_sample="Sample: ./jmeter_run_all.sh saml_redirect_binding /home/ubuntu/scripts/is_540_scripts/saml/SAML2-SSO-RedirectBinding.jmx"
+
+#### sanity check
+if [ -z "$1" ]
+  then
+  	echo "Please add a scenario name for the current test like, eg: saml_redirect_binding"
+	echo "$usage_msg"
+	echo "$usage_sample"
+	exit
+fi
+
+if [ -z "$2" ]
+  then
+        echo "Please provide a valid jmeter script location."
+	echo "$usage_msg"
+        echo "$usage_sample"
+	exit
+fi
+
+time_to_run=10800
+
+is_540_host="idp.wso2.com"
+is_540_host_username="ubuntu"
```

Comment:
Why do we need to specifically say is_540_host
Just use hostname and what is host_username mean ?